apps/cms/pages/templates/a4_candy_cms_pages/home_page.html

```diff
@@ -3,13 +3,20 @@
 
 {% block content %}
     <div class="hompage__wrapper p-3">
-        <div class="row homepage-header">
-            <div class="col-lg-6">
-                <h1 class="homepage-header__teaser ml-md-4">{{ page.teaser|richtext }}</h1>
+        <div class="hompage-header">
```

Comment:
There is a lot of hompage without the e. :)

Reply:
balls! 
Also I am going to add a setting for the footer now so if that goes well i'll try adding one to this and correct the typos